config/assets/default.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
         'public/lib/angular-ui-router/release/angular-ui-router.js',
         'public/lib/angular-ui-utils/ui-utils.js',
         'public/lib/angular-bootstrap/ui-bootstrap-tpls.js',
```

Comment:
Have you actually tried because it seems to work just fine for me. No errors, dropdown working.